src/Security/Handler/AclSecurityHandler.php

```diff
@@ -187,13 +187,6 @@ public function findObjectAcls(\Traversable $oids, array $sids = [])
         return $acls;
     }
 
-    /**
-     * NEXT_MAJOR: change signature to `addObjectOwner(MutableAclInterface $acl, ?UserSecurityIdentity $securityIdentity = null): void`.
```

Comment:
Why you removed all the information?

Reply:
The same comment exists on the interface